aidial_adapter_vertexai/universal_api/errors.py

```diff
@@ -0,0 +1,23 @@
+from typing import Mapping
+
+from aidial_sdk.chat_completion import Choice, Status
+from aiohttp import hdrs
+
+from aidial_adapter_vertexai.llm.exceptions import UserError
+
+
+def report_user_error(
+    choice: Choice, headers: Mapping[str, str], error: UserError
+) -> None:
+    is_chat_usage = headers.get(hdrs.AUTHORIZATION) is not None
```

Comment:
I'm not sure if this is reliable. Maybe the adapter could always do both things?

Reply:
Yes, indeed, I like this.